core/src/main/java/org/apache/brooklyn/core/objs/BasicConfigurableObject.java

```diff
@@ -53,7 +53,7 @@ public BasicConfigurableObject() {
     }
 
     @Override
-    public void injectManagementContext(ManagementContext managementContext) {
+    public void setManagementContext(ManagementContext managementContext) {
```

Comment:
Aries blueprints injects using setter methods. I had this working with CXF resources, where the management context gets injected by the framework. I've since gone on the jersey path, and I'm not sure how to achieve the same with jersey, so I made a proposal to migrate to CXF altogether.